# Service that can be used to control the robot joint positions using the
# panda_control_server. This is done by publishing the joint efforts commands
# to the right topic.

std_msgs/Float64MultiArray joint_positions
std_msgs/Bool wait
---
bool success